pages/common/git-checkout.md

```diff
@@ -6,6 +6,10 @@
 
 `git checkout -b {{branch_name}}`
 
+- Create and switch to a new branch based on an existing remote branch:
+
+`git checkout -b {{branch_name}} {{remote/source_branch}}`
```

Comment:
Usually when we checkout a remote branch, we also want to track it. Should we add that option too or will it be too complicated? @agnivade @sbrl @pxgamer 
```suggestion
`git checkout -b {{branch_name}} {{remote/source_branch}} --track`
```

Reply:
Heyas! Thanks for checking it out so fast. No pun intended 👈👈😎.

My use-case for it was to demonstrate creating a new branch based on a source branch that has a different name, even from a remote. There was some --track/--tracking option mentioned in the other PR I referenced in the description. Not sure. Up to you guys!